integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/convert_currency/ConvertViewFragment.kt

```diff
@@ -477,61 +465,6 @@ class ConvertViewFragment : Fragment(R.layout.fragment_convert_currency) {
         }
     }
 
-    private fun getFiatAmount(balance: String, currencyCode: String): Pair<Fiat?, Coin?>? {
```

Comment:
Some fragments in Coinbase are way too fat. This method can be moved entirely into the ViewModel which means it should be. Generally, if a method doesn't update UI it doesn't belong in the fragment.